Website/data/queries.js

```diff
@@ -164,9 +173,13 @@ async function deposit(userid, account, amount) {
 
         if (update_res.rowCount == 1) {
             await client.query("COMMIT");
+            if(secure){
+                fs.appendFileSync('logs/deposits.log', new Date().toISOString() + ': ' + amount + ' deposit into account '
```

Comment:
You don't have to change it but for future reference i think using string formatting would look neater:
 `${new Date().toISOString()}: ${amount}... `

Reply:
using those special `'s

Reply:
Thanks for the tip! I do appreciate it. I'll update it here before merging for a bit of practice.